README.md

```diff
@@ -81,6 +82,22 @@ docker run -d --restart always --name github-runner \
   myoung34/github-runner:latest
 ```
 
+With the reusage of the registered runner assuming `CONFIGURED_ACTIONS_RUNNER_FILES_DIR=/actions-runner-files` (can be propogated to all other approaches):
+
+```shell
+# per repo
+docker run -d --restart always --name github-runner \
+  -e REPO_URL="https://github.com/myoung34/repo" \
```

Comment:
i think you meant to add a call to `CONFIGURED_ACTIONS_RUNNER_FILES_DIR`  in this

Reply:
Oops, strange. Thanks!

Reply:
See "assuming `CONFIGURED_ACTIONS_RUNNER_FILES_DIR=/actions-runner-files`"
and line 97
`   -v /path/on/host/system:/actions-runner-files`

Reply:
Ok